scalableminds/webknossos
# List of projects with a specific TaskType

This PR adds the possibility to access a list of all projects that contain tasks with a specific taskType.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- go to the Task Types list view
- click on "Projects" of a task type of your choosing
- there should be a list of exactly those projects that contain a task of this type

### Issues:
- fixes #4420

------
